test/cex/cex_nonlinear_arith.c

```diff
@@ -13,8 +16,8 @@ extern int nd_int(void);
 
 int main(int argc, char**argv) {
   int x = nd_int();
-  __VERIFIER_assume (x >= 1);
-  __VERIFIER_assume (x <= 100);  
+  assume (x >= 1);
+  assume (x <= 100);
```

Comment:
I think the problem here is with wrong assume. With opsem2 this example should be solvable.